Airport Ticket Booking/AirportTicketBooking.Test/UserDataHandlerTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AirportTicketBooking.DBHandler;
+using AirportTicketBooking.Enums;
+
+namespace AirportTicketBooking.Test
+{
+    public class UserDataHandlerTests
+    {
+        [Fact]
+        public async void Validate_ShouldReturnFailure_WhenEmailIsNotValid()
```

Comment:
It's recommended to use a public async Task instead. This allows the test runner to await the test method properly.